src/Acuminator/Acuminator.Tests/Sources/Localization/PX1050/LocalizationMethodsWithHardcodedStrings.cs

```diff
@@ -0,0 +1,44 @@
+﻿using PX.Data;
+using PX.Common;
+
+namespace Acuminator.Tests.Sources
+{
+    public class LocalizationWithHardcodedStrings
+    {
+        public string PXMessagesSimple()
+        {
+            string localizedString;
+            localizedString = PXMessages.Localize("Usernames cannot contain commas.");
```

Comment:
@metallmonster @dropsonic  we won't  implement complex scenarios for this diagnostic like:
```
string msg = Localized.Message1;
PXMessages.Localize(msg);
```
or 

```
string msg = Localized.Message1;
if (condition)
   msg += "Spoil localization";

PXMessages.Localize(msg);
```
Is so, then imho we should log it for example in Jira as a low priority task

Reply:
@metallmonster I see such cases sometimes, so probably they should be covered, but not sure if it should be done right now. Please decide for yourself based on implementations cost, or create a new JIRA item.

Reply:
@dropsonic @SENya1990 I'll create an item for that